app/views/components/docs/autocomplete.yml

```diff
@@ -80,3 +94,24 @@ examples:
       selected_options:
           - fr
           - de
+  autocomplete_with_custom_template:
```

Comment:
This one doesn't seem to work in preview :-(

Reply:
Works here https://content-publisher-revie-pr-571.herokuapp.com/component-guide/autocomplete/autocomplete_with_custom_template if you type `Uni` for example

Reply:
Oops - me being dim.